rdmo/projects/assets/js/interview/components/main/Done.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { baseUrl } from 'rdmo/core/assets/js/utils/meta'
+
+import { projectId }  from '../../utils/meta'
+
+import Html from 'rdmo/core/assets/js/components/Html'
+
+const Done = ({ templates }) => {
+
+  const projectUrl = `${baseUrl}/projects/${projectId}/`
+  const answersUrl = `${baseUrl}/projects/${projectId}/answers/`
+
+  return (
+    <>
+      <Html html={templates.project_interview_done} />
+
+      <p>
+        <a href={answersUrl}>{gettext('View answers')}</a>
+      </p>
+
+      <p>
+        <a href={projectUrl}>{gettext('Back to project overview')}</a>
+      </p>
+    </>
+  )
+}
+
+Done.propTypes = {
+  templates: PropTypes.object.isRequired,
+  overview: PropTypes.object.isRequired
```

Comment:
what is this prop `overview`, is it required or needs to be removed here?

Reply:
I think I forgot to remove it. `overview` is the content of the small api endpoint with project title and catalog, displayed on the top right of the interview. should I remove it in `interview-contact-modal`?

Reply:
yes please, thanks!

Reply:
It should now be fixed in https://github.com/rdmorganiser/rdmo/pull/1188.